locales/is-IS/src/structure.ts

```diff
@@ -342,7 +342,8 @@ export default removeUndefinedLocaleResources({
   'panes.document-operation-results.operation-success_discardChanges':
     'Öllum breytingum síðan síðast var birt hefur nú verið hent. Hægt er að endurheimta hentum drögum úr sögunni',
   /** The text when a duplicate operation succeeded  */
-  'panes.document-operation-results.operation-success_duplicate': undefined, // 'The document was successfully duplicated'
+  'panes.document-operation-results.operation-success_duplicate':
+    'Skjalinu var afritað með góðum árangri',
```

Comment:
'Skjalið var afritað með góðum árangri' would be the correct translation

Reply:
```suggestion
    'Skjalið var afritað með góðum árangri',
```